specifications/active/aRMT-1.0.0.yml

```diff
@@ -6,6 +6,46 @@ version: 1.0.0
 assessment_type: QUESTIONNAIRE
 doc: aRMT Questionnaires definition. Includes Personal Health Questionnaire Depression Scale (PHQ-8), Experience sampling method (ESM) and RSES.
 data:
+  - type: THINC-IT
```

Comment:
When updating a specification, a new version (e.g. `1.1.0`) should be created. This may coincide with the new version of the audio plugin (so that both use the same new version number).

Reply:
A corresponding new file name is needed, aRMT-1.1.0.yml.